frontend/javascripts/oxalis/view/td_view_controls.js

```diff
@@ -23,7 +22,7 @@ function TDViewControls({ isRefreshingIsosurfaces, volumeTracing }: Props) {
     }
   }
   return (
-    <ButtonGroup id="TDViewControls">
+    <Space size={0} id="TDViewControls">
```

Comment:
This does not align and unify the buttons's sizes and the spaces don't look pretty. Maybe we can fix this with a css class. One is already in the _tracing_view.less, but that's not aligning and unifying.